.github/workflows/ci.yml

```diff
@@ -20,6 +20,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: ">=14.16.0 14"
+           node-version: ">=14.16.0 14 || >=16.16.0 16"
```

Comment:
This needs to run as a matrix i.e we should run the tests on v14 and on v16
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
@jamime I've added the matrix in but it's added an additional `test` check and I can't work out why, I don't see any syntax issues and haven't been able to find anything that explains why online.

![image](https://user-images.githubusercontent.com/44157880/183691134-18ec5ef7-42d1-49a7-a0f9-83ec6b24d7a6.png)